compose/production/django/Dockerfile

```diff
@@ -33,6 +33,9 @@ RUN chmod +x /start
 RUN chown django /start
 COPY --chown=django:django . /app
 
+RUN mkdir -p /app/staticfiles
+RUN chown django:django /app/staticfiles
```

Comment:
Since nginx and django need to see this, shouldn't it get mounted as a volume in the compose YAML file?